.github/workflows/ci.yml

```diff
@@ -0,0 +1,14 @@
+name: hersh-CI
+on: [push, pull_request]
+jobs:
+  build_and_test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: configure
+        run: cmake -S . -B build
+      - name: build
+        run: cmake --build build -j2
+      - name: test
+        run: ctest --test-dir build
```

Comment:
I think you need `--output-on-failure` otherwise you won't see what went wrong.

Reply:
Fixed.